claragenomics/dl4atac/train.py

```diff
@@ -122,6 +137,6 @@ def train(*, rank, gpu, task, model, train_loader, loss_func, optimizer, pad,
             " Time Taken: {:7.3f}s".format(time.time()-start), color='yellow', rank=rank)
 
     # Time breakdown for the epoch...
-    total_time = time.time() - start
-    remainder_time = total_time - forward_time - backward_time - print_time
+    #total_time = time.time() - start
```

Comment:
@ntadimeti should we delete this or do you want to add them to a debug logger

Reply:
uncommented this.

Reply:
resolving it and adding an issue to get joyjit's input once he is back in office.